configurations/default/env.yml.tmp

```diff
@@ -17,6 +17,11 @@ AWS_API_SERVER: aws-api-id.execute-api.us-east-1.amazonaws.com
 AWS_API_STAGE: stage-name
 DEFAULT_USAGE_PLAN_ID: usage-plan-id
 
+### Optional phone number prefix (defaults to "+1" [North America])
+# PHONE_COUNTRY_PREFIX: "+1"
+### Optional phone number regexp to enable phone number checking.
+# PHONE_NUMBER_REGEXP: ^\(?([0-9]{3})\)?[-. ]?([0-9]{3})[-. ]?([0-9]{4})$
```

Comment:
This should be hardcoded and check for E.164 format

Reply:
Removed in https://github.com/ibi-group/otp-middleware/pull/83/commits/a9386a6e2f46bc9529e14d51ff9e51106ec87ba6